publisher/moloco.js

```diff
@@ -100,9 +100,19 @@ MolocoSDK.prototype.renderAdWithRedirectUrl = function (bannerAd) {
   const originClickUrl = atag.href;
   atag.href = bannerAd.finallandingurl;
 
-  let xhr = new XMLHttpRequest();
-  xhr.open("GET", originClickUrl, true);
-  xhr.send();
+  if(atag.addEventListener) {
+    atag.addEventListener('click', function(){
+      let xhr = new XMLHttpRequest();
```

Comment:
`const` can be used for declaring xhr. (and below)

Reply:
Thank you for the comment!
I modified it and the method name mentioned below on here: fb4b208